src/data/markdown/docs/01 guides/02 Using k6/14 Executors.md

```diff
@@ -0,0 +1,295 @@
+---
+title: Executors
+excerpt: ''
+hideFromSidebar: false
+---
+
+k6 v0.27.0 introduces the concept of _executors_: configurable modes of executing a
+JavaScript function that can model diverse traffic scenarios in load tests.
+
+Multiple executors can be used in the same script, they can be scheduled to run
+parallel or in sequence, and each executor can run a different JS function with
+different environment variables and tags, so this brings a lot of flexibility with
+organizing and modeling testing scenarios.
+
+Existing global execution options such as `vus`, `duration` and `stages` were
+formalized into standalone executors, while new executors were added to support more
+advanced modes of execution. A major benefit of this restructuring is that support
+for new testing scenarios can be added relatively easily to k6, making it very
+extensible.
```

Comment:
Do you mean new "executors"?

Reply:
You mean instead of "new testing scenarios"? Could be, but I wanted to focus on the end-user benefit (being able to model new testing scenarios _with_ new executors) rather than just the addition of a new executor, which is an internal benefit. It's subtle, but I prefer the way it's currently worded. :)

Reply:
I removed this paragraph in 17b520a. It sounded too changelog-y and it's an internal/development benefit, no need to mention it here.